code/modules/admin/verbs/maprotation.dm

```diff
@@ -68,28 +64,56 @@
 			return
 
 		qdel(M)
+		var/config_file = null
+		var/json_value = list()
+		var/config = tgui_alert(usr,"Would you like to upload an additional config for this map?", "Map Config", list("Yes", "No"))
```

Comment:
Have an `if(!config)`, since tgui_alert will return null if the user closes out of the window and it might throw errors. Might not even need to do `var/config = tgui_alert()`, but could also do a switch statement instead. Just be sure to account for users closing the window out lest that cause some fucky behavior.